vmdb/app/models/miq_alert.rb

```diff
@@ -150,12 +150,11 @@ def self.evaluate_hourly_timer
       prof.miq_alerts.each do |a|
         next unless a.enabled? && a.responds_to_events && a.responds_to_events.include?(HOURLY_TIMER_EVENT)
 
-        model = a.db.constantize
-        unless model.name == "Storage" # No clean way to get zone from storage
-          targets += model.includes(:ext_management_system).select { |i| i.my_zone == zone }
-        else
-          targets += model.all
-        end
+        # Zone has accessors to these related resources:
+        # ems_clusters, storages, hosts, ext_management_systems, miq_servers, vms
```

Comment:
This comment is bound to get stale.  Wondering if we need to enumerate them at all.  Perhaps use a phrase like "such as"